src/backend/distributed/worker/worker_create_or_replace.c

```diff
@@ -213,11 +228,22 @@ WorkerCreateOrReplaceObject(List *sqlStatements)
 			return false;
 		}
 
-		char *newName = GenerateBackupNameForCollision(address);
+		Node *utilityStmt = NULL;
+
+		if (GetOnMismatchAction(address) == ON_MISMATCH_DROP)
```

Comment:
minor: We refer the same concept as `Collision` in the following lines with `GenerateBackupNameForCollision `.

Maybe we use similar terminology?